typos.toml

```diff
@@ -0,0 +1,43 @@
+# https://github.com/crate-ci/typos/blob/master/docs/reference.md
+
+[files]
+extend-exclude = [
+  "*.csv",
+  "*.eps",
+  "*.lvt",
+  "*.tlg",
+  "articles/**",
+  "support/texlive.sh",
+  "support/UShyphen.tex",
+  "texmf/**",
+  "xpackages/xinitials/xinitials-test.tex",
+  "xpackages/xlang/**", # german
+  "xpackages/xor/ISSUES.txt",
+  "xpackages/xor/oxford-trial.tex",
+  "xpackages/xor/xo-sample.tex",
+]
+
+[default]
+extend-ignore-re = [
+  ":[NncVvoxefTFpwD]+",
+  "[aA]lph",
+  "\\.ist",
+  "\\(mis\\)use",
+  "\\{ (fo|(03)?BA|Nd) \\}",
+  "\\{inteval\\}",
+  "ABD",
+  "fith",
+  "for emptyness",
+  "IEEE Comput\\.",
+  "juxt(_int|:N)", # special csnames used in l3fp-parse.dtx
+  "LSO",
+  "mis-",
+  "nd",
+  "Noo",
+  "openin",
+  "OT1",
+  "parm",
+  "pre-empted",
```

Comment:
I guess should go for US-style spelling?

Reply:
yes you should I guess, sorry :-)

Reply:
Appending config `locale = "en-us"` would result a looong list of corrections, thus I prefer doing it in a separate PR.

To ask in advance, what's your opinion on renaming `\sys_finalise:` to `\sys_finalize:` (ise -> ize)?